newton/_src/actuators/utils.py

```
@@ -172,22 +186,46 @@ def _load_torch_raw(path: str) -> tuple[Any, dict[str, Any]]:
+        extra_files: dict[str, str] = {_METADATA_FILE: ""}
+        exported = torch.export.load(path, extra_files=extra_files)
+        metadata = json.loads(extra_files[_METADATA_FILE]) if extra_files[_METADATA_FILE] else {}
+        return exported.module(), metadata
+
+    extra_files = {_METADATA_FILE: ""}
     try:
         net = torch.jit.load(path, map_location="cpu", _extra_files=extra_files)
-        meta = json.loads(extra_files["metadata.json"]) if extra_files["metadata.json"] else {}
-        return net, meta
     except Exception:
         pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)